src/why3prop.ml

```diff
@@ -0,0 +1,122 @@
+(** Translation of lambdapi goals to why3 goals in propositional case *)
+
+open Terms
+open Extra
+
+exception NoGoalTranslation
+
+(* a type to present the list of why3 constants and lambdapi terms *)
+type cnst_table = (term * Why3.Term.lsymbol) list
+
+(* number of axioms proved whith the Why3 tactic *)
+let nbr_axioms : int ref = ref 0
+
+(** [get_newname ()] generates a new axiom name. *)
+let get_newname () : string =
+    nbr_axioms := !nbr_axioms + 1;
+    "Why3axiom_" ^ (string_of_int !nbr_axioms)
+
+(* builtins configuration for propositional logic *)
+type prop_config =
+  { symb_P     : sym * pp_hint (** Encoding of propositions.        *)
```

Comment:
Is it useful to keep the pp_hint? Later, you just want to check equality against the first argument. Removing the pp_hint would simplify the code (no call to fst needed anymore).

Reply:
Fixed in 0f3d21044d5f8ef9a6229c0368dd027f30684821